src/content/tutorials/build-a-rust-markdown-parser-webassembly/index.md

```diff
@@ -0,0 +1,156 @@
+---
+updated: 2020-09-12
+difficulty: Advanced
+---
+
+# Build a Markdown Parser with Rust and WebAssembly
+
+This tutorial will walk you through the steps of generating, building, previewing, configuring, and publishing
+a Rust-generated WebAssembly serverless function that parses Markdown for Cloudflare Workers. Let's get started!
+
+This tutorial makes use of [Wrangler](https://github.com/cloudflare/wrangler), our command-line tool for generating, building, and publishing projects on the Cloudflare Workers platform. If you haven't used Wrangler, we recommend [installing Wrangler](/cli-wrangler/install-update), or checking out our [Getting Started guide](/learning/getting-started), which will get you set up with Wrangler, and familiar with the basic commands.
+
+## Generating from template
+
+Cloudflare's command-line tool for managing Workers projects, Wrangler, has great support for templates – pre-built collections of code that make it easy to get started writing Workers. We'll make use of the [rustwasm-worker template](https://github.com/cloudflare/rustwasm-worker-template/) to start building your project.
+
+In the command line, generate your Workers project by passing in a project name (we'll use rustwasm-markdown-parser), and the template URL to base your project on.
+
+```sh
+wrangler generate rustwasm-markdown-parser https://github.com/cloudflare/rustwasm-worker-template/
```

Comment:
Prefix line with at least `$`. See https://docs-engine-docs.adam.workers.dev/markdown#terminals for examples. If it’s a series of steps requiring a specific set of directors, prefix with `~/ $` to start and then after e.g. cd-ing into `rustwasm-markdown-parser`, prefix with `~/rustwasm-markdown-parser $` etc.